apps/src/appMain.js

```diff
@@ -76,7 +76,9 @@ export default function (app, levels, options) {
     blocksCommon.install(Blockly, blockInstallOptions);
     options.blocksModule.install(Blockly, blockInstallOptions);
 
-    if (options.blocksModule.installCustomBlocks && level.customBlocks) {
+    if (options.blocksModule.installCustomBlocks &&
+        level.customBlocks &&
+        !level.hideCustomBlocks) {
       options.blocksModule.installCustomBlocks(
```

Comment:
This will hide the custom block impl from the users, or not add them at all?

Reply:
Whoops, yeah I meant to install them but not add the category. Updated.